Remotion/ObjectBinding/Web/UI/Controls/BocList.cs

```diff
@@ -2383,7 +2401,7 @@ private IEnumerable<BocListRow> GetSelectedRowsInternal ()
         return Enumerable.Empty<BocListRow>();
 
       return _selectorControlCheckedState
-          .Select (rowID => RowIDProvider.GetRowFromItemRowID (Value, rowID))
+          .Select (rowID => RowIDProvider.GetRowFromItemRowID (Value, rowID)!)
```

Comment:
Why the `!`?

Reply:
Linq + NRT aren't working that well. The method returns `IEnumerable<BocListRow>`, the return type would be `IEnumerable<BocListRow?>` without the `!`, `.Where (... != null)` does not influence the type.